docker-compose.yml

```diff
@@ -9,21 +9,32 @@ services:
       - weblate-data:/app/data
     env_file:
       - ./environment
+    ports:
+      - "8000:8000"
     restart: always
     depends_on:
       - database
       - cache
   web:
-    build: nginx
-    links:
-      - weblate
+    image: jwilder/nginx-proxy
+    volumes:
+      - /var/run/docker.sock:/tmp/docker.sock:ro
+      - ./certs:/etc/nginx/certs:ro
+      - /etc/nginx/vhost.d
+      - /usr/share/nginx/html
     ports:
       - "80:80"
-    volumes_from:
-      - weblate:ro
```

Comment:
It is also my understanding that in the Docker world you either have

* Native support for https via letsencrypt in the container deploying the application (that's the case with http://discourse.org for instance)
* A reverse proxy provided with the docker-compose file (that's what people using jrcs/letsencrypt-nginx-proxy-companion do)

Of course Native support is more work and as a result it is not often done this way nor properly maintained. It is much easier (just a few lines as in this commit) to use jrcs/letsencrypt-nginx-proxy-companion.